build.gradle

```diff
@@ -53,6 +53,7 @@ buildscript {
               'design': "com.android.support:design:${versions.supportLibrary}",
               'constraintLayout': 'com.android.support.constraint:constraint-layout:1.1.0-beta4',
           ],
+          'threetenabp': 'com.jakewharton.threetenabp:threetenabp:1.0.5',
```

Comment:
This isn't an Android library so it shouldn't be under 'android', but I'd prefer not to include it altogether: https://github.com/JakeWharton/SdkSearch/pull/38